app/views/dashboard/show.html.erb

```diff
@@ -9,22 +9,29 @@
 <% end %><br><br><br><br>
 <div class="center-align">
 
-	<!-- <a class="waves-effect waves-light btn-small #c5e1a5 light-green lighten-3 black-text">
+
+	<a class="waves-effect waves-light btn-small #c5e1a5 light-green lighten-3 black-text">
 		<center>
 			<%= form_tag "/dashboard", method: :patch do %>
 			<%= label_tag :city %>
 			<%= text_field_tag :city %>
-			<%= submit_tag 'City' %>
+			<%= submit_tag 'Set City' %>
 			<% end %>
 		</center>
 	</a>
-</div> -->
+</div>
 
+<div class="center-align">	<%= form_tag("/restaurant", method: :get, id: 'user_input') do %>
+	<h4>Where are you looking?</h4>
 
-<div class="center-align">
-	<%= form_tag(restaurant_path, method: "get") do %>
+	  <input type="radio" id="current_location" name="location" value="current location class='current_location'" checked><label for="current_location">Use Current Location</label>
```

Comment:
Add two radio buttons to let user decide which location params they would like to be used.